### DISCUSSION TITLE
Load Modeling

### DISCUSSION THREAD

**Discussion by jd-lara** [Sienna-Platform/PowerSystems.jl#846] [Ideas]
We are opening this discussion for the re-design of load modeling in PowerSystems.jl to include: 

~1. Dynamic Modeling of ZIP loads~ 
2. Price Sensitivity of loads. 
~3. Refactor the use of ENUMS for types to dispatch on AC load behavior.~

**Comment by rodrigomha** [comment]
Here is the PSS/E model descriptor of the DER_A model:
https://www.dropbox.com/s/e0hiod6dvwlvjgr/DER_A_PSSE.pdf?dl=0

**Comment by jd-lara** [comment]
The current plan stands as follows: 

Remove the load model enum from the package and add a new struct StandardLoad that is able to model ZIP loads in detail. 

The use of PowerLoad or StandardLoad will be determined at the moment of parsing the data.

**Comment by llavin13** [comment]
A couple thoughts and things that would help me. If the question are dumb no worries, happy to come to a meeting or follow up.

Question(s)

- I don't understand what the load model enum is. Any chance I can get a link to where that is in the code?

Options
1. Bid-in demand with PWL cost function. I think this is what you are proposing in the abstract. Sure, this can be done. We might want to chat with Nongchao about what he's done in the past for other models.
2. Represent demand-side aggregate flexibility as a generator (storage-like). This is closer to what we've done in other modeling w/ PLEXOS and ReEDS. It gives a bit more flexibility in representing aggregations of thousands-to-millions of customers who might be in a "VPP" and is easier to preserve intertemporal links in dispatch that may be relevant if you eventually want to model things like building flexibility. 

Happy to set up a meeting w/ us an maybe Nongchao (I listed various folks on the LDRD as advisors) if it makes sense to discuss.
